index.html

```diff
@@ -0,0 +1,116 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=0.1">
+	<title>Ellie Robinson - Developer</title>
+	<link href="normalise.css" type="text/css" rel="stylesheet"> <!-- this makes sure all browsers start with the same style -->
+	<link href="style.css" type="text/css" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300&display=swap" rel="stylesheet"> 
```

Comment:
This isn't a huge concern but you can put all the fonts in one <link> tag and that way it will only pull them in 1 request not 3 - should be faster page load